modules/OCPPConfiguration/config-mapping.yaml

```diff
@@ -0,0 +1,312 @@
+- component:
```

Comment:
Please add a yaml schema file to validate the format of the mapping file

Reply:
Done